chia/plotting/plot_tools.py

```diff
@@ -316,3 +316,41 @@ def find_duplicate_plot_IDs(all_filenames=None) -> None:
         for filename_str in duplicate_filenames:
             log_message += "\t" + filename_str + "\n"
         log.warning(f"{log_message}")
+
+
+def load_plot(
+    filename: str,
+) ->  Tuple[PlotInfo,str]:
+    if filename==None:
```

Comment:
The `1 for Testing equality to None` warning you got is probably this `==`?

Reply:
Sorry, I'm a new programmer. Thanks for your guidance, I have fixed the format issues.